components/scrapeninja/actions/non-js-scraping/non-js-scraping.mjs

```diff
@@ -0,0 +1,106 @@
+import { ConfigurationError } from "@pipedream/platform";
+import { parseObject } from "../../common/utils.mjs";
+import scrapeninja from "../../scrapeninja.app.mjs";
+
+export default {
+  key: "scrapeninja-non-js-scraping",
```

Comment:
@luancazarine can you rename the actual action names / slugs? Since they are new components I think it'd be best for them to match the name before being shipped

Reply:
The filenames are still the same - maybe the commit wasn't pushed?
`/actions/non-js-scraping/non-js-scraping.mjs`
should match the action name, as in
`/actions/scrape-without-js/scrape-without-js.mjs`
The same goes for the component key, for both components